lib/still/preprocessor/renderer.ex

```diff
@@ -29,7 +29,7 @@ defmodule Still.Preprocessor.Renderer do
           |> Enum.map(&String.replace(&1, "_", ""))
           |> Enum.map(&String.capitalize/1)
 
-        Module.concat([@preprocessor | name])
+        Module.concat(["R#{Enum.random(0..100_000)}" | [@preprocessor | name]])
```

Comment:
What's the reason for this change?

Reply:
@gabrielpoca

Reply:
Via call we concluded this was the same problem redefining modules I was facing before. This is a fair approach for now 👍